tasks/routes/users.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const uuid = require('uuid');
+
+const router = express.Router();
+
+const userStorage = new Map();
+
+function User(login, password, age, uid = uuid.v4()) {
```

Comment:
Consider creating a "models" folder at the root level, next to "routes" folder, and keeping the Users logic and its map there.